source/user_guide.rst

```diff
@@ -0,0 +1,67 @@
+User Guide
+==========
+
+The git packaging workflow is currently an opt-in mechanism for submissions to
+openSUSE:Factory via `src.opensuse.org <https://src.opensuse.org>`_ instead of
+`OBS <https://build.opensuse.org/>`_. It allows packagers to maintain their
+package sources in the `pool <https://src.opensuse.org/pool>`_ organization
+instead of on OBS. Submissions are handled via pull requests on
+https://src.opensuse.org instead of submit requests on OBS.
+
+A bot has been setup that will automatically forward eligible pull requests as
+submitrequests directly to openSUSE:Factory.
+
+
+Packager Guide
+--------------
+
+.. hint::
+   Consider adjusting the package description to alert contributors that
+   ordinary submitrequests are no longer possible and where they should send a
+   pull request instead.
+
+To participate in the test drive, modify the development package in the
+developroject as follows:
+
+1. Login to https://src.opensuse.org using the openSUSE IDP provider.
+2. Checkout the latest state of your develop package from OBS. If it differs
+   from the state of the package in https://src.opensuse.org/pool/$pkg_name,
+   please submit a pull request.
+3. Modify the package config of the devel package via :command:`osc meta -e pkg
+   $develprj $pkgname` by adding the following entry:
+
+.. code-block:: xml
+
+   <package name="$pkg_name" project="$develprj">
+     <!-- snip -->
+     <scmsync>https://src.opensuse.org/pool/$pkg_name#factory</scmsync>
+   </package>
+
+
+Contributor Guide
+-----------------
+
+
+1. Find the package that you want to contribute to in src.opensuse.org/pool and
+   fork it from the repository src.opensuse.org/pool/$pkg_name
+
+2. Clone your fork via :command:`git clone src.opensuse.org/$username/$pkg_name`
+   and implement your changes:
+
+.. code-block:: shell-session
+
+   cd $pkg_name
+   $EDITOR $pkg_name.spec
+   osc vc -m "$CLEVER_CHANGELOG_ENTRY" $pkg_name.changes
+
+3. [optional] Run a local build via :command:`osc
+   build --alternative-project=openSUSE:Factory standard $arch [$SPEC.spec]`.
+
+4. Commit your changes and push them to your fork.
+
+5. Create a pull request against the package in src.opensuse.org/pool.
```

Comment:
However, https://src.opensuse.org/pool/python-pass-git-helper.git has only `devel/` branch? Where does the `factory/` comes in?

I don't think https://src.opensuse.org/pool/python-pass-git-helper/pulls/1 created exactly by this HOWTO will work.

Reply:
that was a setup bug specific to python-pass-git-helper, it has been fixed already.